src/Sulu/Bundle/MediaBundle/Resources/public/scss/modules/media-selection.scss

```diff
@@ -4,201 +4,215 @@
 
 .media-selection-overlay-container,
 .media-selection-overlay {
-  width: 900px !important;
+    width: 900px !important;
+    height: 500px !important;
 
-  @media(max-width: 1024px) {
-    width: 690px !important;
-  }
+    @media(max-width: 1024px) {
+        width: 690px !important;
+    }
 
-  @media(min-width: 1600px) {
-    width: 1325px !important;
-  }
-}
+    @media(min-width: 1600px) {
+        width: 1325px !important;
+    }
 
-.media-selection-overlay-container {
+    @media(min-height: 700px) {
+        height: 600px !important;
+    }
 
-  &.dropzone-overlay-opened {
-    .media-selection-overlay-content {
-      overflow: hidden;
+    @media(min-height: 800px) {
+        height: 700px !important;
     }
-  }
 
-  .slides {
-    margin: 0;
-    padding: 0;
-  }
+    @media(min-height: 900px) {
+        height: 800px !important;
+    }
```

Comment:
Is it necessary to be solved like this? Can't we just put a margin on the overlay, and let it take the rest of the height, instead of redefining it that often?